src/main/java/com/google/cloud/teleport/templates/common/JdbcConverters.java

```diff
@@ -99,6 +103,11 @@
    */
   private static class ResultSetToTableRow implements JdbcIO.RowMapper<TableRow> {
 
+    static SimpleDateFormat df = new SimpleDateFormat("yyyy-MM-dd"); // date format
```

Comment:
Better variable names would be dateFormatter, datetimeFormatter, timestampFormatter

Reply:
done